wasm-rpc/Cargo.toml

```diff
@@ -13,7 +13,6 @@ include = []
 
 [lib]
 path = "src/lib.rs"
-crate-type = ["cdylib", "rlib"]
```

Comment:
This is needed due to https://github.com/rust-lang/cargo/issues/6313#issuecomment-480014371.

Do we have a need for the cdylib currently? If yes, I will check whether I can find a workaround

Reply:
I think it's needed when it is used in guest code (like the generated rpc stubs which we still support, although probably not for long)

Reply:
Ok, will mark as draft while looking for a workaround then

Reply:
You can try it out by creating an app with two components using `static-wasm-rpc` dependency type (or one from the `test-components`) and just tell golem-cli to use the local modified wasm-rpc crate instead of the published one (there is an env var for that)

Reply:
Yes, I tested the non-cdylib with a test-component and it works